Source/Examples/WPF.SharpDX/SimpleDemo/MainViewModel.cs

```diff
@@ -40,15 +40,14 @@ public class MainViewModel : BaseViewModel
         public Color4 DirectionalLightColor { get; private set; }
         public Color4 AmbientLightColor { get; private set; }
 
-
         public MainViewModel()
         {
             // titles
-            this.Title = "Simple Demo";

```

Comment:
@objorke It's your project and I'll play by your rules. But, I would be remiss if I did not recommend you remove this particular rule. By enforcing proper naming of local members, you can make it obvious to the reader of the code that it is a local member, without having to resort to thousands of `this` in the code.